log.c

```diff
@@ -366,6 +367,17 @@ log_redirect_stderr_to(const char *logfile)
 		     strerror(errno));
 		exit(1);
 	}
+#ifdef WINDOWS /* WINDOWS */
+	/*
+	Set the owner of the log file to current user and only grant
+	current user the full control access
+	*/
+	if (set_secure_file_permission(logfile, NULL) != 0) {
```

Comment:
In the set_secure_file_permission(),

Since we are not inheriting the access control entries (ACEs) from the parent directory.. we can specify the dcal as NULL as part of SetNamedSecurityInfoW().. also we can remove DACL_SECURITY_INFORMATION flag ..